### PR TITLE
bug: Correcting URLs on https://rubygems.org/gems/forem_lite

### PR DESCRIPTION
**Pull request by MikeRogers0**
## What is the purpose of this pull request?

Fixes the metadata links on https://rubygems.org/gems/forem_lite

image

## What changes did you make? (overview)

Right now they're linking to `https://github.com/andrewmcodes/forem_lite/forem_lite` instead of `https://github.com/andrewmcodes/forem_lite`. So updated the references in the gemspec.

## Checklist

- [ ] I've added tests for this change
- [ ] I've added a Changelog entry
- [x] I've updated a documentation

If you liked this PR, please consider [buying me a coffee](https://www.buymeacoffee.com/MikeRogers0) ☕️

### CODE REVIEW

File: forem_lite.gemspec [hunk 1]
```diff
@@ -11,11 +11,11 @@ Gem::Specification.new do |s|
...
   s.metadata = {
-    "bug_tracker_uri" => "#{s.homepage}/forem_lite/issues",
+    "bug_tracker_uri" => "#{s.homepage}/issues",
     "changelog_uri" => "#{s.homepage}/blob/main/CHANGELOG.md",
-    "documentation_uri" => "#{s.homepage}/forem_lite",
-    "homepage_uri" => "#{s.homepage}/forem_lite",
-    "source_code_uri" => "#{s.homepage}/forem_lite"
+    "documentation_uri" => s.homepage.to_s,
+    "homepage_uri" => s.homepage.to_s,
+    "source_code_uri" => s.homepage.to_s
```
Comment (andrewmcodes) [inline forem_lite.gemspec]: No need for `.to_s` on these.

**Review by andrewmcodes — CHANGES_REQUESTED** [review]
Lol thanks for catching this. One small request!

File: forem_lite.gemspec (hunk 1 above)
Comment (MikeRogers0) [reply forem_lite.gemspec]: @andrewmcodes Sorted :D

**Review by andrewmcodes — APPROVED** [review]
Thank you!

**Pull request merged**